docs/source/installation/vonBertalanffy.md

```diff
@@ -0,0 +1,125 @@
+# This is an instruction for installing all necessary tools to use vonBertalanffy
+
+## First, begin by installing all the necessary dependencies. At the time of writing this, the following were used:
+
+ * Ubuntu 16.04 LTS
+
+ * Python 2.7
+
+ * NumPy 1.11.1
+
+ * ChemAxon MarvinBeans 17.28.0
+
+ * OpenBabel 2.3
+
+The following commands are entered in a *terminal window (bash or similar  shell)*.
+
+### Python 2
+OpenBabel only works with Python 2. Most distributions should already have
+this installed, but if this is not the case, the following lines will do it:
+```
+ sudo add-apt-repository ppa:fkrull/deadsnakes
+ sudo apt-get update
+ sudo apt-get install python2.7
+```
+Alternatively, Python 2 is freely available at http://www.python.org/download/
+
+### NumPy
+NumPy can be installed using the following commands:
+```
+ sudo apt-get install python-de
```

Comment:
```
pip install numpy
```

Reply:
I have put this command and left the others as an alternative way